src/lib/converter/nodes/module.ts

```diff
@@ -31,7 +31,7 @@ export class ModuleConverter extends ConverterNodeComponent<ts.ModuleDeclaration
         context.withScope(reflection, () => {
             var opt = context.getCompilerOptions();
             if (parent instanceof ProjectReflection && !context.isDeclaration &&
-                (!opt.module || opt.module == ts.ModuleKind.None)) {
+                (!opt.module || (opt.module as any) === ts.ModuleKind.None)) {
```

Comment:
It seems somehow to be needed.... 

`src/lib/converter/nodes/module.ts(34,33): error TS2365: Operator '===' cannot be applied to types 'ModuleKind.CommonJS | ModuleKind.AMD | ModuleKind.UMD | ModuleKind.System | ModuleKind.ES6' and 'ModuleKind.None'.
`